dist/vokal-ng-api.js

```diff
@@ -341,10 +355,31 @@ angular.module( "vokal.API", [ "vokal.Humps" ] )
 
                                 return;
                             }
+                            else
+                            {
+                                $rootScope.$broadcast( "APIRequestUnauthorized", options, data, status );
```

Comment:
Yeah, `=== true` means you want your normal error handling to be interrupted, thus the return.  Same with when you provide a function.  If you don't set it to true, it still broadcasts the unauthorized event, but the lack of a `return` lets it also continue to the promise rejection block later on.